### PR TITLE
feat: add delta w to masterportal

### PR DESCRIPTION
**Pull request by ZenVega**
Import DeltaW layer and adapt styling and legend. Ticket: https://app.asana.com/0/1206402792434175/1207300077590867/f

### CODE REVIEW

**Comment by ZenVega** [comment]
The last commit includes all the prettify updates. Ignore this commit for the review

File: src_3_0_0/modules/abimo/components/AbimoMeasure.vue [hunk 1]
```diff
@@ -161,6 +161,23 @@ export default {
       console.warn(properties);
       console.warn(JSON.stringify(obj));
     },
+    calculatePercentages(feature) {
+      const verdunstun = Math.floor(parseFloat(feature.values_.verdunstun));
```
Comment (LuiseBrandenburger) [inline src_3_0_0/modules/abimo/components/AbimoMeasure.vue]: I would prefer if all values are translated to english (verdunstung -> evaporation).

**Review by LuiseBrandenburger — COMMENTED** [review]
Please translate variables to english

File: src_3_0_0/modules/abimo/components/AbimoMeasure.vue (hunk 1 above)
Comment (ZenVega) [reply src_3_0_0/modules/abimo/components/AbimoMeasure.vue]: yes, hast du völlig recht. die variablen düerften in der nuachsten version von abimo dann auch in english kommen. habs angepasst

**Pull request merged**